Makefile

```diff
@@ -1,9 +1,15 @@
+APIDGOPACKAGES = $(shell go list ./... | grep -v vendor)
+APIDGOFILES = $(shell git ls-files | grep '.go$$' | grep -v vendor)
```

Comment:
Should this be single `$`? `grep '.go$'`?

Reply:
Actual regex for my grep is '.go$' but to escape `$` in make I had to put in `$$`

Reply:
Cool, thanks